corehq/apps/app_manager/templates/app_manager/v1/partials/case_config_advanced.html

```diff
@@ -178,7 +178,7 @@ <h3 class="panel-title" data-bind="html: header"></h3>
                 <div class="form-group">
                     <label>{% trans "Case Type" %}</label>
                     <select class="form-control" data-bind="
-                        options: config.caseTypes,
+                        options: caseConfig.caseTypes,
```

Comment:
finally got around to looking following up on this. Those two examples should be fine as that `config` is a local variable within the `case-config:condition` template. Im open to changing it for consitency's sake but it should work either way.